src/main/webapp/app/tasks/task-list/task-list.component.html

```diff
@@ -1,9 +1,47 @@
 <mat-list data-cy="task-list">
-  <mat-list-item *ngFor="let task of tasks" class="mat-elevation-z1">
-    <mat-icon mat-list-icon>assignment</mat-icon>
-    <h4 mat-line>{{task.name}}</h4>
-    <button mat-icon-button aria-label="Delete task" color="primary">
-      <mat-icon aria-label="Delete task" (click)="delete(task)">delete</mat-icon>
-    </button>
-  </mat-list-item>
-</mat-list>
+  <section *ngIf="pendingTasks?.length>0" class="completed-lists">
+    <div class="wrapper">
+      <h2>To Do</h2>
+    </div>
+    <mat-list-item *ngFor="let task of pendingTasks" class="mat-elevation-z1">
+      <mat-icon mat-list-icon>assignment</mat-icon>
+      <h4 mat-line>{{task.name}}</h4>
+      <h4 mat-line class="done">{{ task.status }}</h4>
+      <button mat-icon-button aria-label="Delete task" color="primary" (click)="delete(task)">
+        <mat-icon aria-label="Delete task">delete</mat-icon>
+      </button>
+      <button mat-icon-button aria-label="Update task status" color="primary" (click)="updateStatus(task,'done')">
+        <mat-icon matListIcon>done</mat-icon>
+      </button>
+      <button mat-icon-button aria-label="Update task status" color="primary"
+        (click)="updateStatus(task,'in progress')">
+        <mat-icon matListIcon *ngIf="task.status==='in progress'" [ngStyle]="{'color':'blue'}">pending</mat-icon>
+        <mat-icon matListIcon *ngIf="task.status!=='in progress'">pending</mat-icon>
```

Comment:
I would probably prefer a dropdown over having so much different icons with regards to changing the same property

Reply:
Updated with a dropdown feature